javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistAnnotationDeclaration.java

```diff
@@ -100,7 +100,7 @@ public boolean isAssignableBy(ResolvedReferenceTypeDeclaration other) {
 
     @Override
     public List<ResolvedReferenceType> getAncestors(boolean acceptIncompleteList) {
-        throw new UnsupportedOperationException();
+        return javassistTypeDeclarationAdapter.getAncestors();
     }
```

Comment:
Would it be possible to have: 

```java
    @Override
    public List<ResolvedReferenceType> getAncestors(boolean acceptIncompleteList) {
        return javassistTypeDeclarationAdapter.getAncestors();
    }
```

If not (or if it's possible but not yet implemented), could we add documentation to explain why?

Reply:
Could you point the difference here please? I'm not understanding it.

Reply:
Oh... silly typo on my part.... doh!

I meant to ask about passing on `acceptIncompleteList` rather than silently dropping it :+1: 

```java
    @Override
    public List<ResolvedReferenceType> getAncestors(boolean acceptIncompleteList) {
        return javassistTypeDeclarationAdapter.getAncestors(acceptIncompleteList);
    }
```

Reply:
.. but it seems you might have dealt with adding that in anyway it seems! :grinning: